docs/content/docs/2.components/slideover.md

```
@@ -197,6 +197,67 @@ slots:
 :placeholder{class="h-full min-h-48"}
 ::
 
+### Size
+
+Use the `size` prop to change the size of the Slideover. Defaults to `md`.
+
+::component-code
+---
+prettier: true
+ignore:
+  - title
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)